README.md

```diff
@@ -18,9 +28,18 @@ Latest snapshot: [![Sonatype Nexus](https://img.shields.io/nexus/s/co.elastic.ot
 Use the `-javaagent:` JVM argument with the path to agent jar.
```

Comment:
sorry, this section looks wrong now, it should just be the download links, no ref to javaagent nor the shell command, my bad I didn't delete thise when I switch to the download links. Need to delete the line and the two triple backticks here

Reply:
Hmm or actually my change was wrong, the markdown when rendered has a #Download and #Run section? I may have messed this up

Reply:
yes, sorry I messed this bit up, need to revert the Run section